cmd/oras/root/cp.go

```diff
@@ -115,58 +117,60 @@ func runCopy(ctx context.Context, opts copyOptions) error {
 	}
 
 	// Prepare copy options
+	started := &sync.Map{}
 	committed := &sync.Map{}
```

Comment:
Do we really need the two variables `started` and `committed`? Maybe one is enough?

Reply:
Since `oras.Copy` will be called more than once, we cannot rely on oras-go to de-duplicate the copying. 

Say we need to copy two blobs `A` and `B` with same digest in difference calls of `oras.Copy` with below time sequence:
1) copy A
2) copy B
3) done copying A
4) done copying B

The logs will be like
```
Copying <DIGEST_OF_A>
Copying <DIGEST_OF_B>
Copied  <DIGEST_OF_A>
# no Copied  <DIGEST_OF_B> since A is commited
```
So we need to add `started` to track whether a copying is started or not, to get rid of the `Copying <DIGEST_OF_B>`.

Reply:
But then do we still need `committed`? Currently `committed` is like, not actively used anywhere.

Reply:
`committed` is used for deduplicating post copy logs. Resolving as the implementation is already changed.